keras_hub/src/utils/preset_utils.py

```diff
@@ -165,6 +166,39 @@ def get_file(preset, path):
                 raise ValueError(message)
     elif scheme in tf_registered_schemes():
         return tf_copy_gfile_to_cache(preset, path)
+    elif scheme == MODELSCOPE_SCHEME:
+        try:
+            from modelscope.hub.snapshot_download import snapshot_download
+        except:
+            raise ImportError(
+                "`from_preset()` requires the `modelscope` package to "
```

Comment:
the error message can modified as follows
    "To load a preset from ModelScope {preset} using `from_preset`, install the "
    "`modelscope` package with: `pip install modelscope`."